base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
I think this API is completely fine, it's just a matter of spelling. Strictly speaking, yes we should use `Pipe` for this since it makes a pipe object.